components/script/dom/webidls/GPUDevice.webidl

```diff
@@ -5,13 +5,14 @@
 // https://gpuweb.github.io/gpuweb/#gpudevice
 [Exposed=(Window, DedicatedWorker)/*, Serializable */, Pref="dom.webgpu.enabled"]
 interface GPUDevice : EventTarget {
-    readonly attribute GPUAdapter adapter;
+    /*[SameObject]*/ readonly attribute GPUAdapter adapter;
     readonly attribute object extensions;
     readonly attribute object limits;
 
+    /*[SameObject]*/ readonly attribute GPUQueue defaultQueue;
```

Comment:
Why are the SameObject annotations commented out?

Reply:
Thank you! These comments was unintendedly left in.